onyo/lib/commands.py

```diff
@@ -717,8 +717,8 @@ def onyo_new(inventory: Inventory,
         table column.
 
     template
-        The name of a template file in ``.onyo/templates/`` that is copied as a
-        base for the new assets to be created.
+        Path to a template file. If relative, this is allowed to be relative to ``.onyo/templates/``.
```

Comment:
or CWD.

Reply:
Nope. CWD is for CLI only. We resolve at the outer layer. Things in lib usually expect absolute paths.